source/localizable/_code-of-conduct.fr.md

```diff
@@ -0,0 +1,99 @@
+Objectif
+-------
+
+Un des objectifs principaux des groupes d'utilisateurs et des conférences se
+basant sur le code de conduite suivant est d'être accessible au plus grand nombre de contributeurs venant des milieux les
+plus divers et variés. Pour ce faire, nous nous engageons à fournir un
+environnement amical, sûr et accueillant pour tous, quel que soit le
+genre, l'orientation sexuelle, le handicap, l'ethnie, la nationalité, le milieu social
+et la religion (ou son absence) des participants.
+
+Ce code de conduite esquisse nos attentes pour tous les membres de la
+communauté, ainsi que les conséquences pour qui y contreviendrait.
+
+Nous invitons tous ceux qui participent à nos événements à nous aider à
+créer un environnement sûr et une expérience positive pour chacun.
+
+
+La citoyenneté Open [Source/Culture/Tech]
+--------------------------------------
+
+Une autre intention de ce code de conduite est d'intensifier le sentiment de citoyenneté Open [Source/Culture/Tech] en encourageant les participants à renforcer les actions dans ce sens et à en reconnaître les bienfaits pour la communauté.
+
+Les communautés reflètent les sociétés dans lesquelles elle existent. Les actions positives sont donc essentielles pour combattre les nombreuses
+formes d'inégalité et d'abus qui existent dans la société.
+
+Si vous observez quelqu'un qui prend un soin particulier à s'assurer que notre communauté est accueillante, amicale et
+encourageante pour tous ses participants, faites le nous savoir.
+
+
+Comportement attendu
+-----------------
+
+* Participez activement et avec authenticité. Ce faisant, vous
+contribuerez à rendre la communauté saine et durable.
+* Faites montre de considération et de respect dans vos paroles et
+ actes.
+* Préférez la coopération au conflit.
+* Ne pratiquez pas le harcèlement, la méchanceté et la discrimination
+ dans votre comportement et dans vos paroles.
+* Soyez attentif à votre environnement et aux autres participants.
+ Alertez les organisateurs si vous êtes témoin d'une situation
+dangereuse, d'une personne en situation de détresse ou d'une violation
```

Comment:
(don't let @skade see this change in leading spaces, nor the change from `*␣` to `␣` instead of `␣␣` 3 lines up :sweat_smile: :trollface: )